.github/workflows/test.yml

```diff
@@ -12,7 +12,37 @@ on:
 jobs:
   run_checker:
     uses: ./.github/workflows/reusable-run-checker.yml
-              
+
+  protos-compiled:
```

Comment:
@joroshiba I am not sure if the `tests` workflow is the right place for the CI job. On the one hand we compile rust code there, on the other this feels more fitting for the `release` workflow.

Reply:
Release in this instance is the push to buf. This seems fine here.